homework/my_first_etl_postgres.py

```diff
@@ -0,0 +1,78 @@
+#Install packages and connect to Postgres
+import dataset
+import sqlalchemy
+import pandas as pd
+
+ds = dataset.connect("postgresql://postgres@localhost/my_first_database")
+
+#Extract tables from Postgres
+engine = sqlalchemy.create_engine("postgresql://postgres@localhost/my_first_database")
+
+customers = pd.read_sql_query('select * from "customers_1"', con=engine)
+employees =pd.read_sql_query('select * from "employees_1"', con=engine)
+offices =pd.read_sql_query('select * from "offices_1"', con=engine)
+orders_composition =pd.read_sql_query('select * from "orders_composition_1"', con=engine)
+orders_delivery = pd.read_sql_query('select *, price_each::numeric as price_each_numeric, order_date::DATE as order_date_datetime from "orders_delivery_1"', con=engine).drop(columns="order_delivery_id")
+product_stock_details =pd.read_sql_query('select *, buy_price::numeric as buy_price_numeric from "product_stock_details_1"', con=engine)
+products =pd.read_sql_query('select * from "products_1"', con=engine)
+
+
+#Transform the data
+
+#Create a megatable
+merge_1= orders_composition.merge(orders_delivery, on= ['order_number', 'price_each'])
+merge_2= merge_1.merge(customers, on= 'customer_number')
+merge_3=merge_2.merge(products, on= 'product_code')
+merge_4=merge_3.merge(product_stock_details, on='product_code')
+merge_5=merge_4.merge(employees, left_on= 'sales_rep_employee_number', right_on= 'employee_number')
```

Comment:
A pythonic way to do this is to just replace the same variable name, thus: 

```python
merge = merge.merge(employees,...)
```

Although, also I would recommend giving it a name other than "merge" or "supermerge", which is really not very descriptive of what's inside, it's descriptive of the process getting to that point, but that's of no interest to anyone using it!